bokehjs/src/coffee/core/util/bbox.coffee

```diff
@@ -5,6 +5,20 @@ export empty = () -> {
   maxY: -Infinity
 }
 
+export positive_x = () -> {
+  minX:  Number.MIN_VALUE,
+  minY: -Infinity,
+  maxX:  Infinity,
+  maxY:  Infinity
```

Comment:
Use trailing comma. This isn't automatically verified, but it's in a good taste (I see where this came from, but still). Or you can use no commas at all (it's allowed by coffeescript).